.travis.yml

```diff
@@ -3,3 +3,10 @@ node_js:
   - "6"
   - "5"
   - "4"
+env:
+  - CC=gcc-4.8 CXX=g++-4.8
+addons:
+  apt:
+    packages:
+      - gcc-4.8
+      - g++-4.8
```

Comment:
I guess this needs everything from https://github.com/Shouqun/node-dbus#dependencies plus `libexpat1-dev`